odoo/tools/translate.py

```diff
@@ -1155,9 +1155,9 @@ def trans_load(cr, filename, lang, verbose=True, create_empty_translation=False,
                                    verbose=verbose,
                                    create_empty_translation=create_empty_translation,
                                    overwrite=overwrite)
-    except IOError:
+    except IOError as io_error:
         if verbose:
-            _logger.error("couldn't read translation file %s", filename)
+            _logger.error("Couldn't read translation file %s. Here's what we got back instead: %s", filename, io_error)
```

Comment:
@mart-e do you mean to replace this line like this?
```suggestion
            _logger.exception("Couldn't read translation file %s", filename)
```

Reply:
Yes, this should be enough no?
But how do you trigger this error anyway? Isn't what is raised in `trans_load_data` enough already?